sentry_sdk/integrations/celery.py

```diff
@@ -46,10 +64,13 @@
 class CeleryIntegration(Integration):
     identifier = "celery"
 
-    def __init__(self, propagate_traces=True):
-        # type: (bool) -> None
+    def __init__(self, propagate_traces=True, monitor_beat_tasks=False):
```

Comment:
Now we need a flag for enabling monitoring of our Beat schedules, because we auto discover the celery app. fyi @sl0thentr0py